crates/syntax/src/ast/node_ext.rs

```diff
@@ -252,7 +252,7 @@ impl ast::Impl {
         }
     }
 
-    pub fn trait_(&self) -> Option<ast::Type> {
+    pub fn target_trait(&self) -> Option<ast::Type> {
```

Comment:
I've meant to go in the opposite direction -- `self_ty` to me feels better that `target_ty`, and, with `self_ty`, `target_trait` doesn't need `targert` )

Reply:
Ahaha, okay, good thing I made that separate commit 😅

Reply:
Was wondering what you meant with item tree and hir def since I didn't have to change anything there